spec/features/collections/collection_details_spec.rb

```diff
@@ -202,6 +202,22 @@
     end
   end
 
+  context 'when selecting a collection with a description with links' do
+    before :all do
+      load_page '/search/collection-details', focus: 'C1200230663-MMT_1', env: :sit, ac: true
+    end
+
+    it 'displays the external link' do
+      expect(page).to have_content('Greenland Ice Mapping Project (GIMP)')
+      # expect(page).to have_link("Greenland Ice Mapping Project (GIMP)", href: "http://nsidc.org/data/measures/gimp")
```

Comment:
Let's keep new PRs as clean as possible by removing debug or unused code.

Reply:
I pushed this so that you guys could pull my branch and help debug my test in reference to my email to the team - in the future should I not push commits unless I know they work? Do I not need to push my commits in order for others to pull them locally?